internal/testhelper/wftest/cr.go

```diff
@@ -2,20 +2,16 @@ package wftest
 
 import (
 	wf "github.com/wavefrontHQ/wavefront-operator-for-kubernetes/api/v1alpha1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type CROption func(*wf.Wavefront)
-
-func CR(options ...CROption) *wf.Wavefront {
+func CR(options ...func(*wf.Wavefront)) *wf.Wavefront {
 	cr := &wf.Wavefront{
-		TypeMeta: v1.TypeMeta{},
-		ObjectMeta: v1.ObjectMeta{
+		ObjectMeta: metav1.ObjectMeta{
 			Name:      "wavefront",
-			Namespace: "testNamespace",
+			Namespace: DefaultNamespace,
 		},
 		Spec: wf.WavefrontSpec{
-			Namespace:            "testNamespace",
```

Comment:
Good catch!

Reply:
Previously, that was because our test setup functions were only accepting`wf.WavefrontSpec`. I've been wanting to kill that from the moment that I wrote it. It was just too big for the ticket we were working on.